tetrateci/tsb-util/tsb_util.py

```diff
@@ -411,11 +415,23 @@ def main():
     if args.password is not None:
         password = args.password
 
+    provider = "others"
+    if args.aws is True:
+        provider = "aws"
+
     certs.create_root_cert()
 
     os.mkdir("generated")
 
-    for tenant in range(configs.tenant_count):
+    tenant_count = 0
+
+    for conf in configs.app:
+        for replica in conf.replicas:
+            if replica.tenant_id > tenant_count:
```

Comment:
use the max function -- tenant_count = max(tenant_count, tenant_id). tenant_id in the config file need not be in order.